Zend/zend_API.c

```diff
@@ -2473,6 +2475,12 @@ ZEND_API void zend_unregister_functions(const zend_function_entry *functions, in
 		fname_len = strlen(ptr->fname);
 		lowercase_name = zend_string_alloc(fname_len, 0);
 		zend_str_tolower_copy(ZSTR_VAL(lowercase_name), ptr->fname, fname_len);
+		reg_function = zend_hash_find_ptr(target_function_table, lowercase_name);
+		if (reg_function->common.arg_info &&
```

Comment:
This should also have a `reg_function &&` chechk, because zend_unregister_functions is also called in error cases where not all functions have been defined, for example: https://github.com/devnexen/php-src/blob/f3ee93de6574c479515519c4e49271777f383264/Zend/zend_API.c#L2236